lib/ecto_job/worker.ex

```diff
@@ -32,7 +32,9 @@ defmodule EctoJob.Worker do
   @spec log_duration(EctoJob.JobQueue.job(), DateTime.t()) :: :ok
   defp log_duration(_job = %queue{id: id}, start = %DateTime{}) do
     duration = DateTime.diff(DateTime.utc_now(), start, :microseconds)
-    Logger.info("#{queue}[#{id}] done: #{duration} µs")
+    if Application.get_env(:ecto_job, :disable_logging, false) do
```

Comment:
I'd prefer not to have the double-negative of "disable false" here.

I think this reads a bit better for me:
```elixir
if Application.get_env(:ecto_job, :log, true) do
```

Another slightly more complex variant would be to allow configuring the level:

```elixir
Logger.log(Application.get_env(:ecto_job, :logger_level, :info), "#{queue}[#{id}] done: #{duration} µs")
```
[Plug.Logger](https://hexdocs.pm/plug/Plug.Logger.html#module-options) does something similar. Just an idea.